server/server.go

```diff
@@ -53,6 +54,7 @@ func New(c *Config) (*Server, error) {
 		c.Github,
 		githubapp.WithClientUserAgent(userAgent),
 		githubapp.WithClientMiddleware(
+			githubapp.ClientCaching(func() httpcache.Cache { return httpcache.NewMemoryCache() }),
```

Comment:
We'll have to watch the memory usage of this after deployment. I'm not sure how much memory will actually be used by cached requests over the lifetime of the service, so we may eventually want something like https://github.com/die-net/lrucache instead.